### PR TITLE
Now fixing bug mentioned in groups

### PR DESCRIPTION
**Pull request by Skeeve**
See https://groups.google.com/forum/#!topic/tiddlywiki/FQBDk0pAJEE

### CODE REVIEW

File: core/modules/filters/x-listops.js [hunk 1]
```diff
@@ -23,13 +23,22 @@ Extended filter operators to manipulate the current list.
         return results;
     };
 
+    /*
+    Return the number the str represents
+    Return default if it's not a number
+    */
+    function getInt( str, deflt ) {
```
Comment (Jermolene) [inline core/modules/filters/x-listops.js]: House style is not to have spaces in parameter lists: `function getInt(str,deflt) {`

File: core/modules/filters/x-listops.js [hunk 2]
```diff
@@ -23,13 +23,22 @@ Extended filter operators to manipulate the current list.
...
+    	var i= parseInt( str );
```
Comment (Jermolene) [inline core/modules/filters/x-listops.js]: We should provide 10 as the radix parameter: `var i = parseInt(str);`

Also note that house style is that there should be a space on either side of the "=" of an assignment

File: core/modules/filters/x-listops.js (hunk 1 above)
Comment (Jermolene) [inline core/modules/filters/x-listops.js]: This function should go in `$:/core/modules/utils/utils.js`

File: core/modules/filters/x-listops.js [hunk 3]
```diff
@@ -23,13 +23,22 @@ Extended filter operators to manipulate the current list.
...
+    }
+
     /*
     Moves a number of items from the tail of the current list before the item named in the operand
     */
     exports.putbefore = function (source, operator) {
         var results = prepare_results(source),
             index = results.indexOf(operator.operand),
-            count = parseInt(operator.suffix) || 1;
+            count = getInt(operator.suffix, 1);
```
Comment (Jermolene) [inline core/modules/filters/x-listops.js]: Again, there shouldn't be a space after the comma (and for the subsequent occurances)

File: core/modules/filters/x-listops.js (hunk 2 above)
Comment (Skeeve) [reply core/modules/filters/x-listops.js]: 0 is not required as it is the default, isn't it?

Did it nonetheless.

**Comment by Jermolene** [comment]
Thanks @Skeeve

**Pull request merged**